functions/handlers/books.js

```diff
@@ -65,7 +67,7 @@ exports.editBook = (req, res) => {
     desc: req.body.desc,
     title: req.body.title,
     userHandle: req.user.handle,
-    imageUrl: req.user.imageUrl,
```

Comment:
I think you should still leave the userImage in here just in case we need to display their picture on the edit page

Reply:
I did not complete to uploading book image yet. I am still working with this :)

Reply:
Alrighty : )